ui/src/config/section/compute.js

```diff
@@ -108,6 +108,7 @@ export default {
           docHelp: 'adminguide/virtual_machines.html#changing-the-vm-name-os-or-group',
           dataView: true,
           popup: true,
+          show: (record) => { return record.vmtype !== 'storagefsvm' },
```

Comment:
Should we hide the options or should we show some warning in the dialogs? I don't have a strong preference though

Reply:
These operations are not allowed via API call as well, as they can unpredictably affect file share function. So I think it's better if they are not shown in UI as well.

Reply:
These options are not allowed via apis calls as well as they can affect file share function unpredictably, so I think it's better to hide them from ui as well.